arches_lingo/src/arches_lingo/api.ts

```diff
@@ -16,11 +16,11 @@ export const login = async (username: string, password: string) => {
         body: JSON.stringify({ username, password }),
     });
     try {
-        const parsed = await response.json();
+        const responseJson = await response.json();
```

Comment:
these names got changed as I was exploring earlier solutions with attempting to access session cookies. Happy to revert if need be /shrug